marklogic-data-hub-central/ui/src/components/flows/flows.tsx

```diff
@@ -716,27 +807,29 @@ const Flows: React.FC<Props> = (props) => {
         formData.append("files", file);
       });
 
-      if (!runFlowClicked) {
-        await props.runStep(runningFlow, runningStep, formData)
-          .then(resp => {
-            setShowUploadError(true);
-            setFileList([]);
-          });
-      } else {
-        let stepNumbers = [{}];
-        for (let i = 0; i < selectedStepDetails.length; i++) {
-          stepNumbers.push(selectedStepDetails[i]);
-        }
-        stepNumbers.shift();
-        await props.runFlowSteps(runningFlow, stepNumbers, formData)
-          .then(resp => {
-            setShowUploadError(true);
-            setFileList([]);
-            setSelectedStepOptions({});
-            setSelectedStepDetails([{stepName: "", stepNumber: -1, stepDefinitionType: "", isChecked: false}]);
-            setRunFlowClicked(false);
-          });
-      }
+      // if (!runFlowClicked) {
```

Comment:
These commented lines are part of another story?

Reply:
Yes @aniamy , Is part of the previous story, right now I'm not using this lines but could be necessary in the next pr, if not, at the end i will erase it!.